CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+
+## Welcome to the MyLibreLab project!
+
+First off, thank you for considering contributing to MyLibreLab. It's people like you that make open source possible! :tada:
+
+MyLibreLab is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into MyLibreLab itself.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+# Ground Rules
+
+Responsibilities
+* Ensure that code that goes into core meets all requirements in this checklist: {{CHECKLIST}}
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds.
+
+
+# Your First Contribution
+
+Unsure where to begin contributing to MyLibreLab? You can start by looking through these labels!
+* good first issue - issues which should only require a few lines of code, and a test or two.
+* help wanted  - issues which should be a bit more involved than good first issues.
+
+Working on your first Pull Request? You can check these resources:
+* http://makeapullrequest.com/
+* http://www.firsttimersonly.com/
+
+
+#### At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first :smile_cat:
+
+
+# Getting started
+
+1. Create your own fork of the code.
+2. Run the tests. We only take pull requests with passing tests, and it's great to know that you have a clean slate.
+3. Each pull request should be clear, concise and informative: Please consider checking these resources: https://www.pullrequest.com/blog/writing-a-great-pull-request-description/ and https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests
+4. Each pull request should implement ONE feature or bugfix. If you want to add or fix more than one thing, submit more than one pull request.
+5. Each commit name and message should be clear, concise and informative: Please consider checking these resouces:https://chris.beams.io/posts/git-commit/ and https://dev.to/chrissiemhrk/git-commit-message-5e21
+6. Each commit should be small, contained and should solve only ONE problem.
+
+
+# Community
+
+You can chat with the MLL users and devs in here! {{Discord}}
+
+#### If you find a security vulnerability, do NOT open an issue. Email marko.dev@pm.me instead.
```

Comment:
There is a separate template for security vulnerabilities in the repository settings.